lib/pacemaker/pcmk_sched_actions.c

```diff
@@ -144,7 +144,8 @@ action_uuid_for_ordering(const char *first_uuid, const pe_resource_t *first_rsc)
         if (pcmk_is_set(first_rsc->flags, pe_rsc_notify)
             && ((first_rsc->parent == NULL)
                 || (pe_rsc_is_clone(first_rsc)
-                    && (first_rsc->parent->variant == pe_container)))) {
+                    && (first_rsc->parent->variant == pcmk_rsc_variant_bundle))
+               )) {
```

Comment:
do we state that explicitly in the guidelines? both alignment and maybe as well
when we decide not to close everything in the last line ... 
at least for my understanding this should be aligned with the block it closes.

Reply:
no, this was just a really hard to follow condition. i think it'll be better to separate a bool variable for part of the condition

Reply:
i believe if pe_rsc_notify is set and parent is NULL, that first_rsc has to be a clone, so we can move pe_rsc_is_clone(first_rsc) to the outer conditions

Reply:
and at that point, parent == NULL or variant_bundle is redundant, since a clone has to meet one of those two conditions

Reply:
apparently the bundle resource itself can have pe_rsc_notify set and hit parent==NULL, so a bool variable it will be

Reply:
pe_rsc_is_clone(first_rsc) || pe_rsc_is_bundled(first_rsc)) doesn't break any regression tests, so I think that covers it